### PR TITLE
new release_04.00.01

### PR DESCRIPTION
**Pull request by MariePrandstetter**
Issue : #5059

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 4.1.0 / 2021-10-29
```
Comment (PhilippeAccorsi) [inline changelog.md:1]: wrong version number

Comment (PhilippeAccorsi) [reply changelog.md:1]: and date

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+
+### Fixed Issues
+
+- Backend: [#5048](https://github.com/tracim/tracim/issues/5048)
+
```
Comment (PhilippeAccorsi) [inline changelog.md:6]: Please add one more space between two version.

**Review by PhilippeAccorsi — CHANGES_REQUESTED** [review]

**Review by grignards — APPROVED** [review]

**Review by PhilippeAccorsi — APPROVED** [review]

**Comment by PhilippeAccorsi** [comment]
@MariePrandstetter Can you add an issue related to this version ?

**Comment by PhilippeAccorsi** [comment]
Also: if you add fixed issues in this version before merge, you need to change the date of the release.

File: changelog.md [hunk 3]
```diff
@@ -3,6 +3,9 @@
 ### Fixed Issues
 
 - Backend: [#5048](https://github.com/tracim/tracim/issues/5048)
+- Frontend: [#5049](https://github.com/tracim/tracim/issues/5049)
+[#4895](https://github.com/tracim/tracim/issues/4895)
```
Comment (PhilippeAccorsi) [inline changelog.md:7]: Missing `,` at end of first issue
ordering of issue is not good

**Review by PhilippeAccorsi — CHANGES_REQUESTED** [review]

**Review by PhilippeAccorsi — APPROVED** [review]

**Pull request merged**